PowerShell/ScubaGear/baselines/teams.md

```diff
@@ -3,12 +3,11 @@
 
```

Comment:
For [Teams policy group 6 license requirements](https://github.com/cisagov/ScubaGear/blob/1393-update-scbs-with-new-front-matter-text-approved-by-occ/PowerShell/ScubaGear/baselines/teams.md#license-requirements-5), need to add the "However, this requirement can also be met through third-party services." (or similar language as you see fit) to indicate 3rd party service usage.

"DLP for Teams requires an E5 or G5 license. See [Microsoft Purview Data Loss Prevention: Data Loss Prevention for Teams | Microsoft Learn](https://learn.microsoft.com/en-us/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance/microsoft-365-security-compliance-licensing-guidance#microsoft-purview-data-loss-prevention-data-loss-prevention-dlp-for-teams) for more information. However, this requirement can also be met through third-party services."

Reply:
updated using similar language used for AAD in your first comment.